Misc/NEWS.d/next/Build/2018-12-05-22-28-40.bpo-35257.dmcd_s.rst

```diff
@@ -0,0 +1,2 @@
+Avoid leaking the linker flags into distutils when compiling
+C extensions.
```

Comment:
You should mention LTO compilation with clang. Maybe: "Avoid leaking Link Time Optimization (LTO) flags in linker flags into distutils when compiling C extensions."

Reply:
Noted, will do.

Reply:
LTO is targeted now as the one that shouldn't propagate to C extensions, however that should target the all the flags that are not intended for propagation to C extensions, so shouldn't we have a more generic description of it?

Reply:
> so shouldn't we have a more generic description of it?

Keep the generic description, but add a sentence to explain in which case the bug was the most visible. For example:

"Avoid leaking the linker flags into distutils when compiling C extensions, especially Link Time Optimization (LTO) flags."

From what I see in this PR, LDFLAGS="$LDFLAGS $LTOFLAGS" replaced with LDFLAGS_NODIST="$LDFLAGS_NODIST $LTOFLAGS" is the major change of this PR. I don't see other linker flags only passed to LDFLAGS_NODIST. Maybe during the temporary profiling step for PGO? But that's not visible for users.

Reply:
Addressed.